conf/k8s.yml

```diff
@@ -0,0 +1,66 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations: {}
+  labels:
+    app: ${REPO_NAME}
+  name: ${REPO_NAME}
+  namespace: ${NAMESPACE}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ${REPO_NAME}
+  template:
+    metadata:
+      labels:
+        app: ${REPO_NAME}
+    spec:
+      containers:
+      - env:
+        - name: MYSQL_DB_HOST
+          value: 127.0.0.1:3306
+        - name: MYSQL_DB_USERNAME
+          valueFrom:
+            secretKeyRef:
+              key: username
+              name: cloudsql-db-credentials
+        - name: MYSQL_DB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              key: password
+              name: cloudsql-db-credentials
+        image: gcr.io/${GOOGLE_PROJECT}/${REPO_NAME}:${COMMIT}
+        imagePullPolicy: Always
+        name: ${REPO_NAME}
+        ports:
+        - containerPort: 8080
+        volumeMounts:
+        - mountPath: "/orsp-config"
+          name: orsp-config
+      - command:
+        - "/cloud_sql_proxy"
+        - "-instances=orsp-dev:us-east1:bricc-mysql-dev=tcp:3306"
```

Comment:
I think this should be injected based n environment.

Reply:
Options: instance name may be the same if we have a database per environment in same mysql instance. Or we may have N mysql instances, one per environment,  with just one database (same name) per instance. Not sure what may be the best option to choose. Thoughts ?

Reply:
> Options: instance name may be the same if we have a database per environment in same mysql instance. Or we may have N mysql instances, one per environment, with just one database (same name) per instance. Not sure what may be the best option to choose. Thoughts ?

My suggestion is to have completely separate database instances, each uniquely named, one per cluster that is configured just for that cluster. If we want to have more envs in the project, we can, and each one can point to their own db instance. This way, you can have a `dev` cluster, a `dgil-dev` cluster for your own dev work, a `grushton-dev` cluster for my own work, etc. It also frees us up to have a `staging` or `perf` environment in the same google project.

In that world, the database name is a configuration and not hard-coded.

Reply:
This is still an outstanding problem. The db name should be parameterized somehow.

Reply:
I think this the last config change pending. I am on, working on proper syntax.